Server/DB/Scripts/rooms/addUserToRoom.go

```diff
@@ -0,0 +1,39 @@
+package roomsScripts
+
+import (
+	"context"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"ishtaloo.io/DB/Collections"
+	entities "ishtaloo.io/Entities"
+)
+
+func AddUserToRoom(ctx context.Context, roomId string, user *entities.User) {
```

Comment:
Why is ctx passed?

Reply:
there are two DB operations in this function, one to find a room and another to preform an update, both require ctx